executor/analyze.go

```diff
@@ -118,7 +118,15 @@ func (e *AnalyzeExec) Next(ctx context.Context, req *chunk.Chunk) error {
 	// The meaning of key in map is the structure that used to store the tableID and indexID.
 	// The meaning of value in map is some additional information needed to build global-level stats.
 	globalStatsMap := make(map[globalStatsKey]globalStatsInfo)
-
+	finishJobWithLogFn := func(ctx context.Context, job *statistics.AnalyzeJob, meetError bool) {
+		job.Finish(meetError)
+		if job != nil {
+			logutil.Logger(ctx).Info("analyze finished",
```

Comment:
Could we log more fields to indicate which analyze this job belongs to?
Or maybe the field `start time` can be removed. (let me involve more people to discuss) /cc @qw4990 @eurekaka @zz-jason

Reply:
Yes, we can log any information stored in instance of `executor#analyzeResult` and `statistics#AnalyzeJob`. The final implementation depending on feature requests.

Reply:
How about writing a proposal to discuss this feature? This PR solves another issue, IMO, this PR can be merged now.

Reply:
@winoros I agree with @rebelice. On the hand our discussing may take a while.

Reply:
I suggest that we at least also print `DB`, `Table/Partition`, `Col/Index` fields here, otherwise this log seems meaningless since users don't know which job it belongs to, isn't it?

Reply:
@qw4990 I think it's ok. how about that guys? @rebelice @winoros

Reply:
@hidehalo I apologize for the late reply. I agree with this point, I think this PR can be merged after adding these fields.

Reply:
@rebelice It's OK.